### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by raffo1234**
- This PR creates elements for login and sign up pages.
- Url: `/login`
- Use `react-hook-form` for the form 
- Use `yup` for validations
- Expected:
loginPage

### CODE REVIEW

File: src/components/LoginPage/LoginPage.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import VisibilityToggle from "components/shared/VisibilityToggle";
+import AlertMessage from "components/shared/AlertMessage";
+import Stack from "components/shared/Stack";
+import Button from "components/shared/Button";
+import FormLabel from "components/shared/FormLabel";
+import logotipo from "images/logotipo.png";
+import styles from "./LoginPage.module.css";
+
+const SignupSchema = yup.object().shape({
+  email: yup.string().required("Email es required").email("Incorrect email"),
```
Comment (hpneo) [inline src/components/LoginPage/LoginPage.js:18]:
```suggestion
  email: yup.string().required("Email is required").email("Incorrect email"),
```

**Comment by alessandrapereyra** [comment]
Nice. Any reason for `react-hook-form` over Formik?

File: src/components/PlaygroundPage/PlaygroundPage.js [hunk 2]
```diff
@@ -169,8 +169,8 @@ const PlaygroundPage = () => (
       </CircularButton>
       <Card />
       <Field placeholder="eg... http:zombo.com" />
-      <Field icon={DashboardIcon} placeholder="eg... http:zombo.com" />
-      <Field icon={DashboardIcon} text="EST" placeholder="Time" />
+      <Field leftIcon={DashboardIcon} placeholder="eg... http:zombo.com" />
```
Comment (alessandrapereyra) [inline src/components/PlaygroundPage/PlaygroundPage.js:172]:
```suggestion
      <Field leftIcon={DashboardIcon} placeholder="eg... http://zombo.com" />
```

Although we should probably replace instances of zombo.com with example.com or something like that.

Comment (raffo1234) [reply src/components/PlaygroundPage/PlaygroundPage.js:172]: Yeah, also it is just the `playground` page used only for examples.

**Comment by raffo1234** [comment]
@alessandrapereyra I don't have a strong option about `Formik` or `react-hook-form`. I just thinking in hooks 👍

**Pull request merged**